saltapi/service/proposal_service.py

```diff
@@ -374,6 +375,18 @@ def update_proprietary_period(
             proposal_code=proposal_code, proprietary_period=proprietary_period
         )
 
+        release_date = self.repository.get_release_date(proprietary_period, proposal_code)
+        body = {
+            "query": "mutation($proposalCode:String!,$institution:Institution!,$apiKey:String!,$releaseDate:String!){updateReleaseDates(proposalCode:$proposalCode,institution:$institution,apiKey:$apiKey,metadataReleaseDate:$releaseDate,dataReleaseDate:$releaseDate){status}}",
+            "variables": {
+                "proposalCode": proposal_code,
+                "institution": "SALT",
+                "releaseDate": str(release_date),
+                "apiKey": get_settings().ssda_api_key
+            }
+        }
+        requests.post(get_settings().ssda_api_url, json=body)
```

Comment:
You aren't handling errors.

Reply:
i need what ever error the SSDA raises wich will course 500 internal server error whic exceptions_handles take care of